test/sdf/pose_1_9.sdf

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" ?>
+<sdf version="1.9">
+  <world name="default">
+    
+    <model name="model_with_empty_pose">
+      <pose></pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_pose_value">
+      <pose>1 2 3 1 2 3</pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_empty_translation">
+      <pose>
+        <translation></translation>
+      </pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_translation_value">
+      <pose>
+        <translation>2 3 4</translation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_empty_rpy_deg">
+      <pose>
+        <rotation type="rpy_degrees"></rotation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_rpy_deg_value">
+      <pose>
+        <rotation type="rpy_degrees">0 90 180</rotation>
```

Comment:
I would be careful using pitch values of 90 degrees in an example since that is a singularity